### ISSUE TITLE
Depends on NPM version the testPathPattern doesn't work sometime

### ISSUE THREAD

**Issue by budnix** [handsontable/handsontable#7411]
### Description

The `testPathPattern` argument doesn't work for different NPM versions. The NPM populates (depends on version) the env variables as `npm_config_testPathPattern` or `npm_config_testpathpattern`. This causes the issue.

### Steps to reproduce

1. Run the unit tests selectively by typing `npm run test:unit --testPathPattern=cells`;
2. It doesn't work in node 15 (node+npm), in Node 12 does.

### Your environment
* Handsontable version: -
* Browser Name and version: -
* Operating System: -

**Comment by swistach** [comment]
Maybe we should consider removing `--testPathPattern` in favour of passing path directly to `jest`?
```
npm run test:unit -- [args][path]
```

**Comment by budnix** [comment]
The `testPathPattern` argument was introduced to support the same pattern as exist in the E2E tests https://handsontable.com/docs/8.2.0/tutorial-testing.html. It's convenient to have a common argument for each test environment.

**Comment by swistach** [comment]
Yeah, I remember it, but it doesn't work on my local env with Win10 and node12+.
I didn't try to run it on a lower version, but 12.x, 14.x and the latest 15 ignore `testPathPattern` for unit tests.

**Comment by budnix** [comment]
That's why the issue was created ;)
